src/pages/dar_collection_review/AtAGlance.js

```diff
@@ -0,0 +1,150 @@
+    fontSize: '1.6rem',
+    fontWeight: 400,
+    display: 'flex',
+    padding: '1rem 2%',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
+  columnStyle: Object.assign({}, Styles.TABLE.HEADER_ROW, {
+    justifyContent: 'space-between',
+  }),
+  cellWidth: {
+    dataUse: '20%',
+    dacFinalDecision: '20%',
+    structuredResearchPurpose: '15%',
+    dacResearchPurposeAgreement: '17.5%',
+    duosAlgorithmDecision: '12.5%',
+    dacVsDuosAlgorithm: '10%'
+  },
+  color: {
+    dataUse: '#000000',
+    dacFinalDecision: '#000000',
+    structuredResearchPurpose: '#000000',
+    dacResearchPurposeAgreement: '#000000',
+    duosAlgorithmDecision: '#000000',
+    dacVsDuosAlgorithm: '#000000'
+  },
+  fontSize: {
+    dataUse: '1.6rem',
+    dacFinalDecision: '1.6rem',
+    structuredResearchPurpose: '1.6rem',
+    dacResearchPurposeAgreement: '1.6rem',
+    duosAlgorithmDecision: '1.6rem',
+    dacVsDuosAlgorithm: '1.6rem'
+  }
+};
+
+export default function AtAGlance(props) {
+  const [collection, setCollection] = useState(props.collection);
+  const [dataUseBuckets, setDataUseBuckets] = useState(props.dataUseBuckets);
+
+  const columnHeaderFormat = {
+    dataUse: {
+      label: 'Data Use',
+      cellStyle: {width: styles.cellWidth.dataUse}
+    },
+    dacFinalDecision: {
+      label: 'DAC Final Decision',
+      cellStyle: {width: styles.cellWidth.dacFinalDecision}
+    },
+    structuredResearchPurpose: {
+      label: 'Structured Research Purpose',
+      cellStyle: {width: styles.cellWidth.structuredResearchPurpose}
+    },
+    dacResearchPurposeAgreement: {
+      label: 'DAC Research Purpose Agreement',
+      cellStyle: {width: styles.cellWidth.dacResearchPurposeAgreement}
+    },
+    duosAlgorithmDecision: {
+      label: 'DUOS Algorithm Decision',
+      cellStyle: {width: styles.cellWidth.duosAlgorithmDecision}
+    },
+    dacVsDuosAlgorithm: {
+      label: 'DAC vs DUOS Algorithm',
+      cellStyle: {width: styles.cellWidth.dacVsDuosAlgorithm}
+    },
+
+    headerCell: {
+      label: 'Header Cell',
+      cellStyle: {width: styles.cellWidth.header},
+    },
+  };
+
+  //todo: use collection and dataUseBuckets to get necessary data
+  //todo: update logic in AtAGlanceCellData.js
+  const columnHeaderData = () => {
+    const {dataUse, dacFinalDecision, structuredResearchPurpose, dacResearchPurposeAgreement, duosAlgorithmDecision, dacVsDuosAlgorithm} = columnHeaderFormat;
+    return [dataUse, dacFinalDecision, structuredResearchPurpose, dacResearchPurposeAgreement, duosAlgorithmDecision, dacVsDuosAlgorithm];
+  };
+
+  const processAtAGlanceData = ({collection, dataUseBuckets}) => {
+    const rowElementMaxCount = 11;
+    //chunking elements subdivides the entire bucket list to rows of max rowElementMaxCount size
+    //Needed to provide predictable design, can't have a flexbox row of n elements with infinitesimal width
+    const chunkedBuckets = chunk(rowElementMaxCount)(dataUseBuckets);
```

Comment:
You don't need to use chunkedBuckets or rowElementMaxCount for this function. These lines were only used for the Vote Summary Header (the bar on top that shows the bucket name and the vote result in a colored box) so that there would be a fixed number of data buckets per row. So if there were more than 10 dataUses, chunking would subdivide the elements into rows of 10 max.

With the table width isn't an issue since the columns are fixed and length is a non-factor, so you can just pass the buckets as is.